search/indexing_api.py

```diff
@@ -109,6 +109,9 @@
     "image_src": {"type": "keyword"},
     "topics": {"type": "keyword"},
     "offered_by": {"type": "keyword"},
+    "created": {"type": "date"},
+    "default_search_priority": {"type": "integer"},
+    "minimum_price": {"type": "scaled_float", "scaling_factor": 10},
```

Comment:
I think the `scaling_factor` needs to be `100` for prices based on [this page](https://www.elastic.co/guide/en/elasticsearch/reference/master/number.html) in the docs. Can you update here and below?

Reply:
Done!